### ISSUE TITLE
Does not work with PHP 8.x

### ISSUE THREAD

**Issue by kbalog** [bpiwowar/papercite#156]
Apparently, the issue is accessing strings with curly braces: https://wordpress.org/support/topic/papercite-error-php8/

**Comment by hansoleorka** [comment]
Yes, replacing the {0} with [0] on lines 179, 191, 248, and 253 in BibTex_osbib.php fixed this issue!

**Comment by dewagter** [comment]
The papercite version distributed at https://wordpress.org/plugins/papercite/ is still version *0.5.18* which does not work anymore with PHP8.0

I made a fix here: https://github.com/digfish/papercite/pull/1

How can this be re-integrated?
